libsolidity/analysis/DeclarationContainer.cpp

```diff
@@ -137,13 +137,25 @@ vector<Declaration const*> DeclarationContainer::resolveName(ASTString const& _n
 
 vector<ASTString> DeclarationContainer::similarNames(ASTString const& _name) const
 {
-	static size_t const MAXIMUM_EDIT_DISTANCE = 2;
+
 	// because the function below has quadratic runtime - it will not magically improve once a better algorithm is discovered ;)
 	// since 80 is the suggested line length limit, we use 80^2 as length threshold
 	static size_t const MAXIMUM_LENGTH_THRESHOLD = 80 * 80;
 
 	vector<ASTString> similar;
-
+	static size_t MAXIMUM_EDIT_DISTANCE = 0;
```

Comment:
This should not be `static` anymore. Also, it should use camelCase now. Finally, it is probably better to use something like `maximumEditDistance = _name.size() > 3 ? 2 : _name.size() / 2;`

Reply:
Thats smart!
Updated